### PR TITLE
Proposal for Expanding Robust Toolbox Modularity

### PR DESCRIPTION
**Pull request by Jezithyr**
Draft PR for my proposal for making Robust Toolbox more modular.
The concept for this is similar to how unreal engine handles plugins:
https://docs.unrealengine.com/5.3/en-US/plugins-in-unreal-engine/
With elements of unity's package manager and held together with duct-type by Ike's GameHub proposal #133 

This is a draft PR, it is not really ready for discussion yet, but feel free to discuss it a bit if you think you have some ideas.
Also, I'm not super sold on Community Modules, they feel like they are effectively just content assemblies.

### CODE REVIEW

File: src/SUMMARY.md [hunk 1]
```diff
@@ -140,6 +139,7 @@ Design Proposals
 - [Exterminator](en/proposals/deltanedas-exterminator.md)
 - [Power Generation Pattern](en/proposals/tday93-power-generation.md)
 - [Turf War](en/proposals/deltanedas-turf-war.md)
+- [Anomaly Cores](en/proposals/jezithyr-expanding-rt-modularity.md)
```
Comment (deltanedas) [inline src/SUMMARY.md:142]: Anomaly Cores :trollface:

Comment (Jezithyr) [reply src/SUMMARY.md:142]: goddamit

**Comment by Jezithyr** [comment]
what the fuuuuck git. I'm going to reopen this as a draft lol

**Comment by Jezithyr** [comment]
wait what the fuck, this force pushed over this... Gah git why....

**Comment by Jezithyr** [comment]
Updated PR is here: #152

**Pull request closed without merging**